### ISSUE TITLE
[BUG] MacOS pkg fails, cask no longer a sub command

### ISSUE THREAD

**Issue by kaushiksachin** [saltstack/salt#60645]
**Description**
While installing package in mac os Catalina. Salt executes "/usr/local/bin/brew cask list --versions".  which fails since cask is not a sub-command anymore . https://github.com/Homebrew/discussions/discussions/902

Command I ran --> /usr/local/bin/salt-call state.highstate --local

**Steps to Reproduce the behavior**
[DEBUG   ] Could not LazyLoad cmd.mod_run_check: 'cmd.mod_run_check' is not available.
[INFO    ] Executing command 'test -f /usr/local/bin/brew' as user 'admin' in directory '/Users/admin'
[DEBUG   ] Last command return code: 0
[INFO    ] ['unless condition is true']
[INFO    ] Completed state [bash install.sh] at time 10:25:44.121793 (duration_in_ms=3059.198)
[DEBUG   ] Could not LazyLoad pkg.ex_mod_init: 'pkg.ex_mod_init' is not available.
[INFO    ] Running state [Install_macos_base_pkg] at time 10:25:44.135826
[INFO    ] Executing state pkg.installed for [Install_macos_base_pkg]
[DEBUG   ] Could not LazyLoad pkg.resolve_capabilities: 'pkg.resolve_capabilities' is not available.
[INFO    ] Executing command 'brew --prefix' in directory '/Users/admin'
[INFO    ] Executing command '/usr/local/bin/brew info --json=v1 --installed' as user 'admin' in directory '/Users/admin'
[INFO    ] Executing command 'brew --prefix' in directory '/Users/admin'
[INFO    ] Executing command '/usr/local/bin/brew cask list --versions' as user 'admin' in directory '/Users/admin'
[ERROR   ] Command '['su', '-l', 'admin', '-c', 'cd -- /Users/admin && { /usr/local/bin/brew cask list --versions\n }']' failed with return code: 1
[ERROR   ] stderr: Error: Unknown command: cask
[ERROR   ] retcode: 1
[DEBUG   ] Could not LazyLoad pkg.normalize_name: 'pkg.normalize_name' is not available.
[INFO    ] All specified packages are already installed

**Versions Report**
<details><summary>salt --versions-report</summary>
(Provided by running salt --versions-report. Please also mention any differences in master/minion versions.) 

```
bash-3.2# salt-call --versions-report
Salt Version:
          Salt: 3002.2
 
Dependency Versions:
          cffi: 1.12.2
      cherrypy: 17.4.1
      dateutil: 2.8.0
     docker-py: Not Installed
         gitdb: 2.0.6
     gitpython: 2.1.15
        Jinja2: 2.10.1
       libgit2: Not Installed
      M2Crypto: Not Installed
          Mako: 1.0.7
       msgpack: 1.0.0
  msgpack-pure: Not Installed
  mysql-python: Not Installed
     pycparser: 2.19
      pycrypto: Not Installed
  pycryptodome: 3.9.8
        pygit2: Not Installed
        Python: 3.7.4 (default, Nov 16 2020, 16:27:58)
  python-gnupg: 0.4.4
        PyYAML: 5.3.1
         PyZMQ: 18.0.1
         smmap: 3.0.2
       timelib: 0.2.4
       Tornado: 4.5.3
           ZMQ: 4.3.1
 
System Versions:
          dist: darwin 19.6.0 
        locale: UTF-8
       machine: x86_64
       release: 19.6.0
        system: Darwin
       version: 10.15.7 x86_64
```
</details>

**Additional context**

**Comment by lashomb** [comment]
This was fixed in 3003[3003](https://docs.saltproject.io/en/latest/topics/releases/3003.html)

**Comment by sagetherage** [comment]
@lashomb thank you!! yes this is fixed here: #58381
